lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts

```diff
@@ -90,6 +90,7 @@ export class DropdownCloudWidgetComponent extends WidgetComponent implements OnI
                 .pipe(takeUntil(this.onDestroy$))
                 .subscribe((result: FormFieldOption[]) => {
                     this.field.options = result;
+                    this.field.updateForm();
```

Comment:
we already have another place where we call the updateForm is not enough ?
```suggestion
                     if (this.isLinkedWidget()) {
            this.loadFieldOptionsForLinkedWidget();
```

Reply:
The ability to add a default value has been removed in modeling, so here we are fixing only a scenario for old apps that contain the default value in their form definition. In the next release we are also changing the validation to not allow you to have a form with a dropdown and a default value https://alfresco.atlassian.net/browse/AAE-6250 Not sure at this point if we need this fix at all

Reply:
Ok what we are missing here is the main point : A dropdown with a value set by a variable not a default value needs to add itself to the form values. If it doesn't add its value to the form values you cannot complete the task .

Reply:
@mauriziovitale It looks like that, expecially 'cause if you don't have the options loaded like in a rest dropdown you don't save any values there.

Reply:
Ok cool, the only point that I see an improvement here is in case the dropdown is linked and of type REST and the form is saved/completed, the updateForm will be called twice. Once here https://github.com/Alfresco/alfresco-ng2-components/blob/cd795c1344c2d6a964446f92ed5815d47f887cc7/lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts#L93 and once here https://github.com/Alfresco/alfresco-ng2-components/blob/cd795c1344c2d6a964446f92ed5815d47f887cc7/lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts#L111

Can we check if there is any way to avoid that double call?

Reply:
If the task is saved/completed and you reload your browser, you will see that it is called twice

Reply:
@arditdomi it's behind an if what IS called twice ? The options retrieval?

Reply:
@arditdomi ok i got it :D 
`The call is better than anything`

Reply:
We had a call with Vito and figured it out. Resolving the comment